index.html

```diff
@@ -11,6 +11,17 @@
 <body>
     <!-- header -->
     <!-- hero -->
+    <!-- creativity -->
+    <!-- intelligence -->
+    <!-- achievements -->
+    <!-- servicies -->
+    <!-- projects -->
+    <!-- updates -->
```

Comment:
between updates and works shoul be "clients" section, the same will go to css and js comments

Reply:
Done